apps/parsley/src/components/SectionsFeatureModal/index.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "@emotion/styled";
+import MarketingModal from "@leafygreen-ui/marketing-modal";
+import { useMatch } from "react-router-dom";
+import { useSectionsFeatureDiscoveryAnalytics } from "analytics/sectionsFeatureDiscovery/useSectionsFeatureDiscoveryAnalytics";
+import routes from "constants/routes";
+import { zIndex } from "constants/tokens";
+import { useSectionsFeatureDiscoveryContext } from "context/SectionsFeatureDiscoveryContext";
+import { useParsleySettings } from "hooks/useParsleySettings";
+import { releaseSectioning } from "utils/featureFlag";
+import { graphic } from "./graphic";
+
+export const SectionsFeatureModal = () => {
+  const { closeFeatureModal, isOpenFeatureModal } =
+    useSectionsFeatureDiscoveryContext();
+  const isViewingTaskLog = useMatch(routes.evergreenLogs);
+  const { sendEvent } = useSectionsFeatureDiscoveryAnalytics();
+  const { updateSettings } = useParsleySettings();
+
+  return releaseSectioning && isViewingTaskLog ? (
+    <StyledMarketingModal
+      buttonText="Enable sectioning"
+      data-cy="sections-feature-modal"
+      graphic={graphic}
+      graphicStyle="center"
+      linkText=""
+      onButtonClick={() => {
+        closeFeatureModal();
+        updateSettings({
+          sectionsEnabled: true,
+        });
+        sendEvent({
+          name: "Clicked feature modal enable sections buttons",
+          release: "beta",
+        });
+      }}
+      onClose={() => {
+        closeFeatureModal();
+        sendEvent({
+          name: "Clicked sections feature modal cancel button",
+          release: "beta",
+        });
+      }}
+      open={isOpenFeatureModal}
+      showBlob
+      title="Introducing Task Log Sectioning Beta Preview!"
+    >
+      <p>
+        Parsley now supports sectioning for <b>task logs!</b> You can now group
+        your task logs into sections to make them easier to navigate and
```

Comment:
Minor copy suggestion since users aren't really responsible for the grouping
```suggestion
        Parsley now supports sectioning for <b>task logs!</b> Evergreen organizes your logs by function and command to make them easier to navigate and
```